nemo_rl/models/generation/sglang/sglang_worker.py

```
@@ -1,805 +1,741 @@
-            gpu_id = self.server_args.base_gpu_id + i
-            uuid = get_device_uuid(gpu_id)
-            gpu_uuids.append(uuid)
+        self.process = p
 
-        return gpu_uuids
+        if self.node_rank == 0 and self.router_ip and self.router_port:
+            payload = {
+                "url": self.server_base_url,
+                "worker_type": "regular",
+            }
+            response = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)